### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by gjsieben**
Updates the README, which still has the original _Dash to Dock_ extension text.

The other small change is setting the "All Apps" button to appear on the top instead of the bottom, similar to old Ubuntu.

### CODE REVIEW

**Comment by gjsieben** [comment]
@didrocks Does this look OK?

File: schemas/10_ubuntu-dock.gschema.override [hunk 1]
```diff
@@ -10,4 +10,5 @@ custom-theme-running-dots-border-color='#e95420'
 extend-height=true
 opaque-background=true
 background-opacity=0.70
+show-apps-at-top=true
```
Comment (didrocks) [inline schemas/10_ubuntu-dock.gschema.override]: We are not going to do this. The app icon is different from the unity bfb. In addition to that, the activities one is really close to it and can be mislead if the 2 actions are too close one from each other. Furthemore, we want to keep as many similarities with GNOME upstream design. That's why this setting wasn't in the first set.

File: README.md [hunk 2]
```diff
@@ -1,16 +1,18 @@
-# Dash to Dock
+# Ubuntu Dock
 ![screenshot](https://github.com/micheleg/dash-to-dock/raw/master/media/screenshot.jpg)
 
-## A dock for the GNOME Shell
+## A dock for Ubuntu
```
Comment (didrocks) [inline README.md]: An Ubuntu dock for the GNOME Shell rather?

File: README.md [hunk 3]
```diff
@@ -1,16 +1,18 @@
...
 This extension enhances the dash moving it out of the overview and transforming it in a dock for an easier launching of applications and a faster switching between windows and desktops without having to leave the desktop view.
 
-For installation instructions and more information visit [https://micheleg.github.io/dash-to-dock/](https://micheleg.github.io/dash-to-dock/).
+For installation instructions and more information visit [https://github.com/ubuntu/dash-to-dock/](https://github.com/ubuntu/dash-to-dock/).
```
Comment (didrocks) [inline README.md]: Please reference rather the ubuntu branch in the upstream dash to dock tree for now. This is where the main reference will leave.

File: README.md [hunk 4]
```diff
@@ -1,16 +1,18 @@
...
+
+This dock is a fork of the original [Dash to Dock](https://github.com/micheleg/dash-to-dock) extension for GNOME. The original extension was very powerful, but we wanted a simpler, more integrated experience for general Ubuntu users. You can install the original extension for more (but not officially supported) features.
 
 ## Installation from source
 
 The extension can be installed directly from source, either for the convenience of using git or to test the latest development version. Clone the desired branch with git
 
-<pre>git clone https://github.com/micheleg/dash-to-dock.git</pre>
+<pre>git clone https://github.com/ubuntu/dash-to-dock.git</pre>
```
Comment (didrocks) [inline README.md]: Same, needs changes to pull the ubnutu branch from dash-to-dock upstream repo

File: schemas/10_ubuntu-dock.gschema.override (hunk 1 above)
Comment (gjsieben) [inline schemas/10_ubuntu-dock.gschema.override]: Okay, I will remove that. I was doing it partially because of feedback from this:

http://www.omgubuntu.co.uk/2017/08/new-ubuntu-dock-first-look

Still, maybe it could be a setting in the new Control Panel...

Comment (gjsieben) [inline schemas/10_ubuntu-dock.gschema.override]: Unless Ubuntu doesn't want that. I couldn't find "I tried to explain though what we don't really want for settings, I hope that makes sense!" Was it a separate article, or were you referring to your comments on the PR?

**Comment by gjsieben** [comment]
README is looking pretty good now. I removed the image on top because it was no longer an accurate representation, and updated the git commands to download the ubuntu-dock branch from micheleg. I also added a section informing users why some features were removed, such as the settings panel.

I removed the automatic "All Apps" button on top as requested.

**Comment by gjsieben** [comment]
A few grammar tweaks later, now it is ready. I also updated the URL in Debian "control" so that GitHub will automatically load the proper branch from micheleg.

File: schemas/10_ubuntu-dock.gschema.override (hunk 1 above)
Comment (didrocks) [inline schemas/10_ubuntu-dock.gschema.override]: Actually, that was my comment. But I'll publish a blog post tomorrow to explain why.

File: README.md [hunk 5]
```diff
@@ -1,38 +1,46 @@
...
+
+This dock is a fork of the original [Dash to Dock](https://github.com/micheleg/dash-to-dock/) (master Branch) extension for GNOME. The original extension was very powerful, but we wanted a simpler, more integrated experience for general Ubuntu users. You can install the original extension for more (but not officially supported) features.
 
 ## Installation from source
 
 The extension can be installed directly from source, either for the convenience of using git or to test the latest development version. Clone the desired branch with git
 
-<pre>git clone https://github.com/micheleg/dash-to-dock.git</pre>
-or download the branch from github. A simple Makefile is included. Then run
+<pre>git clone -b ubuntu-dock https://git@github.com/micheleg/dash-to-dock.git --single-branch</pre>
```
Comment (didrocks) [inline README.md]: You can remove the `-- single-branch` here, correct?

File: README.md [hunk 6]
```diff
@@ -1,38 +1,46 @@
...
+
+Then run
 <pre>make
 make install
 </pre>
-to install the extension in your home directory. A Shell reload is required <code>Alt+F2 r Enter</code> and the extension has to be enabled  with *gnome-tweak-tool* or with *dconf*.
+to install the extension in your home directory. A Shell reload is required <code>Alt+F2 r Enter</code> and the extension has to be enabled  with *gnome-tweak-tool* or with *dconf*. Of course, Ubuntu 17.10 and up have the extension pre-installed.
+
+## A note on Settings
+You may notice that this release is very similar to the original [Dash to Dock](https://github.com/micheleg/dash-to-dock/) but with less features, no settings panel, and different defaults. This is on purpose, because we wanted to bring a simpler dock experience into GNOME Shell for Ubuntu users, and we really didn't need all of the settings that the original Dash to Dock had. It is one of those things where less is more, as most users just don't need (or want) all those settings.
```
Comment (didrocks) [inline README.md]: Can you rephrase slightly? "didn't need all of the settings" -> "didn't want to expose all of the settings" (the settings are still there, can be edited via dconf-editor or other tools, just not exposed graphically).

Also, remove the last sentence, it's not needed in a README IMHO and the blog post I'll publish tomorrow try to convey that.

File: README.md [hunk 7]
```diff
@@ -1,38 +1,46 @@
...
+
+Some settings will be available through an add-on to the Settings program for Ubuntu, but not all. For those who do want all the available options, the original Dash to Dock extension is fully compatible with this dock. Simply install it from [GNOME Extensions](https://extensions.gnome.org/) like normal.
```
Comment (didrocks) [inline README.md]: Maybe direct link to dash-to-dock on GNOME extensions website?

File: README.md [hunk 8]
```diff
@@ -1,38 +1,46 @@
-# Dash to Dock
-![screenshot](https://github.com/micheleg/dash-to-dock/raw/master/media/screenshot.jpg)
-
-## A dock for the GNOME Shell
+## A Ubuntu Dock for the GNOME Shell
```
Comment (didrocks) [inline README.md]: Only one `#`, as it's the title.

File: README.md (hunk 5 above)
Comment (gjsieben) [inline README.md]: I left that there because without it, it will download all the other branches that aren't related to ubuntu-dock. If that is there, it will download the ubuntu-dock branch and the parts of master needed, but none of the other stuff.

The -b ubuntu-dock command only sets the system to use the ubuntu-dock branch after downloading absolutely every branch in the repo.

Comment (didrocks) [inline README.md]: That makes sense thus ;)

**Comment by gjsieben** [comment]
Made all changes except for removing the --single-branch switch. This is because without it, it will download every branch in the tree, resulting in a ton of branches that we don't need. With the --single-branch, it will download ubuntu-dock and the parts of master required, and that's it.

Of course, if you still don't want --single-branch, then that is OK. You also have editing permission if you wanted to drop that quick, then merge (before I can attend to it).

**Comment by didrocks** [comment]
LGTM, thanks a lot for those changes! Merging :)
